src/Jackett.Common/Indexers/IPTorrents.cs

```diff
@@ -31,15 +31,15 @@ public class IPTorrents : IndexerBase
             "https://www.iptorrents.com/",
             "https://iptorrents.me/",
             "https://nemo.iptorrents.com/",
-            "https://ipt.getcrazy.me/",
```

Comment:
Please shift all the links you want to remove from the alternativelinks section to the bottom of the legacylinks section.
So that any users that are currently on one of the legacylinks will have their config automatically changed to the default sitelink.
thanks.

Reply:
My bad. I have done so now. I completely missed it earlier.
Kindly merge if all good now.